apps/balancer-tools/src/app/apr/(utils)/tokenApr.ts

```diff
@@ -55,14 +62,14 @@ export async function getPoolTokensAprForDateRange(
             return {
               address: tokenAddress,
               symbol,
-              yield: await getAPRFromRateProviderInterval(
-                rateProviderAddress as Address,
-                startAt,
-                endAt,
+              yield: await getAPRFromRateProviderInterval({
+                rateProviderAddress: rateProviderAddress as Address,
```

Comment:
what do we gain by enforcing this here or instead of the function accepting a string instead?